src/ttn/thethingsnetwork-send-v1.cpp

```diff
@@ -114,13 +114,13 @@ void onEvent(ev_t ev)
 // used to read data from python program
 void readData()
 {
-    char buf[16];
+    char buf[51];
```

Comment:
Please handle the error when our data is larger than 51 characters.

Reply:
Throw an exception

Reply:
will create new issue for exception handling